gcloud/streaming/http_wrapper.py

```diff
@@ -157,7 +173,7 @@ def length(self):
         Returns:
           Response length (as int or long)
         """
-        def ProcessContentRange(content_range):
+        def _process_content_range(content_range):
```

Comment:
That's fine too. Closures always bug me when the indents start to pile up. They also seem like bug magnets.